app/controllers/terms_controller.rb

```diff
@@ -35,4 +36,11 @@ def render_404
   def vocabulary
     @vocabulary ||= Vocabulary.find(params[:vocabulary_id])
   end
+
+  def authorize
```

Comment:
We should find a way to not duplicate this logic between terms/vocabularies controller.